src/core.js

```diff
@@ -31,6 +31,7 @@ exports.moveTraces = Plotly.moveTraces;
 exports.purge = Plotly.purge;
 exports.setPlotConfig = require('./plot_api/set_plot_config');
 exports.register = Plotly.register;
+exports.toImage = require('./plot_api/to_image');
```

Comment:
makes sense, and I am sure it would get used.  I'll just need to add some tests to feel good about it if it is an "official" function.